tests/stubtest_allowlists/py3_common.txt

```diff
@@ -774,3 +774,7 @@ pkgutil.ImpLoader.get_source
 pkgutil.ImpLoader.is_package
 pkgutil.ImpLoader.load_module
 pkgutil.ImpLoader.source
+
+# We lie about the existence of these methods
+.*.__buffer__
+.*.__release_buffer__
```

Comment:
Not thrilled about this, as it means our CI won't fail if we have a typo in a `__buffer__` method that actually exists at runtime. But I guess I can live with it!

Reply:
Would be nice if the stubtest allowlist would only silence issues about the method not existing, not issues about the signature if it does exist.

Reply:
I guess what we're saying is we're both +1 for https://github.com/python/mypy/issues/13703 :)